docs/nodes/nodes-library/nodes/GoogleSheets/README.md

```diff
@@ -34,39 +34,137 @@ The final workflow should look like the following image.
 
 The Start node exists by default when you create a new workflow.
 
-
+::: v-pre
 ### 2. Set node
 
-1. Click on the ***Add Value*** button and select 'Number' from the dropdown list.
-2. Enter `id` in the ***Name*** field.
-3. Click on the ***Add Value*** button and select 'String' from the dropdown list.
-4. Enter `name` in the ***Name*** field.
-5. Enter the value for the name in the ***Value*** field.
-6. Click on ***Execute Node*** to run the node.
-
-![Using the Set node to set data to be inserted by the Google Sheets node](./Set_node.png)
+The example workflow uses the Set node to generate data that we want to add to the Google Sheet. We can use other nodes, like [HTTP Request node](../../core-nodes/HTTPRequest/README.md), to get the data that we want to add to the Google Sheet.
+
+1. Click on the ***Add Value*** button and select 'String' from the dropdown list.
+2. Enter `Name` in the ***Name*** field.
+3. Enter the name of the house in the ***Value*** field.
+4. Click on the ***Add Value*** button and select 'String' from the dropdown list.
+5. Enter `Rent` in the ***Name*** field.
+6. Enter the rent of the house with the currency (example $1000) in the ***Value*** field.
+7. Click on the ***Add Value*** button and select 'String' from the dropdown list.
+8. Enter `City` in the ***Name*** field.
+9. Enter the name of the city in the ***Value*** field.
+10. Click on the ***Add Value*** button and select 'Number' from the dropdown list.
+11. Enter `id` in the ***Name*** field.
+12. Click on the gears icon next to the ***Value*** field click on ***Add Expression***.
+13. Paste the following expression `{{Math.floor(Math.random()*1000)}}`. This expression will generate a three-digit random number.
+14. Toggle ***Keep Only Set*** to true.
+15. Click on ***Execute Node*** to run the node.
+:::
 
```

Comment:
Can you please add an explanation about what the readers are expected to see in the screenshot below?

Reply:
@tanay1337 will this suffice `In the screenshot below, you can see the output of the Set node.` or do we need a more detailed explanation?

Reply:
I'd also describe a bit about what they can see in the output. In this case, they see that the node has created data based on their settings of the Set node that will be passed in the workflow to the next nodes. Also, they can see that your expression has generated a random three digit number.